src/components/SetupTab/ArgInput/index.jsx

```diff
@@ -8,6 +9,29 @@ import Row from 'react-bootstrap/Row';
 import InputGroup from 'react-bootstrap/InputGroup';
 import Modal from 'react-bootstrap/Modal';
 
+/**
+ * Filter a message that refers to many spatial inputs' bounding boxes.
+ *
+ * Messages are oneline strings with `|` separating each
+ * filename & bounding-box pair. E.g:
+ *
+ *   `Bounding boxes do not intersect:
+ *   ./a.shp: [-84.9, 19.1, -69.15, 29.5] |
+ *   ./b.shp: [-79.0198, 26.481, -78.3717, 27.268] | ...`
+ *
+ * @param {string} message - a string that contains `filepath` e.g:
+ * @param {string} filepath - string preceding the relevant part of `message`
+ * @returns {string} - the filtered and formatted part of the message
+ */
+function filterSpatialOverlapFeedback(message, filepath) {
+  const newPrefix = 'Bounding box does not intersect at least one other:';
+  const bbox = message.split(`${filepath}:`).pop().split('|')[0];
+  const bboxFormatted = bbox.split(' ').map(
+    (str) => str.padEnd(22, ' ')
```

Comment:
Nice, I was wondering about how the padding would be implemented!  How did you arrive at 22, and what will happen if we overrun the 22-character padding here?

Reply:
I had some "real" sample data coordinates with ~18 digits, so 22 allowed for those and added some extra space. If the input string is longer than the pad length, it's returned unchanged. 

I also thought about first reducing the precision to 7 or 8 places, then padding, for even better readability. But I didn't end up bothering with that. I still think it's a good idea since these coords are always returned in decimal degrees, so there's tons of false precision here. But maybe it's better to make that change in `validation.py`, if we want to.